client/app/views/styles/_konnector.styl

```diff
@@ -39,6 +39,10 @@
             margin 30px 0 10px 0
             button
                 outline none
+                &.disable
```

Comment:
is the `disable` class only used here? If yes, then prefer styling `&[aria-busy=true]` for cursor and `&[aria-disabled=true]` for greying. It then do not rely on a class that duplicates native state-support provided by aria.